parcels/examples/example_stommel.py

```diff
@@ -88,6 +94,7 @@ def stommel_example(npart=1, mode='jit', verbose=False, method=AdvectionRK4, gri
     class MyParticle(ParticleClass):
         p = Variable('p', dtype=np.float32, initial=0.)
         p_start = Variable('p_start', dtype=np.float32, initial=fieldset.P)
+        age = Variable('age', dtype=np.float32, initial=0.)
```

Comment:
`float32` should be sufficient for this case study. Generally, `age` shall be the 64-bit float as is the `time` variable of a particle.